src/components/Auth/SignInForm/SignInForm.jsx

```diff
@@ -0,0 +1,40 @@
+import { useInput } from 'hooks/useInput';
+import * as Styled from './SignInForm.styled';
+
+const SignInForm = ({ submitHandler }) => {
+  const [email, onChangeEmail, isValidEmail] = useInput({
+    initialValue: '',
+    minLength: 1,
+    type: 'email',
+  });
+
+  const [password, onChangePassword, isValidPassword] = useInput({
+    initialValue: '',
+    minLength: 8,
+  });
+
+  return (
+    <Styled.SignLayoutWrapper onSubmit={e => submitHandler(e, email, password)}>
+      <h3>로그인</h3>
+      <Styled.InputWrapper>
+        <Styled.Input type="text" value={email} onChange={onChangeEmail} />
+        {!isValidEmail && email.length > 0 && <p>@을 포함해 주세요.</p>}
+      </Styled.InputWrapper>
+      <Styled.InputWrapper>
+        <Styled.Input type="password" value={password} onChange={onChangePassword} />
+        {!isValidPassword && password.length > 0 && <p>8자리 이상 입력해 주세요.</p>}
+      </Styled.InputWrapper>
+      {isValidEmail && isValidPassword ? (
+        <Styled.SubmitBtn type="submit" isAllow={true}>
+          로그인
+        </Styled.SubmitBtn>
+      ) : (
+        <Styled.SubmitBtn isAllow={false} disabled>
+          로그인
+        </Styled.SubmitBtn>
```

Comment:
불 필요한 중복 코드 인거 같아요 아래와 같이 써볼 수도 있을거 같습니다!
```suggestion
      const isValidForm = isValidEmail && isValidPassword;
      return (
        // ...중간 코드
        <Styled.SubmitBtn type="submit" isAllow={isValidForm} disabled={isValidForm}>
          로그인
        </Styled.SubmitBtn>
        // ... 이외 코드
```

Reply:
생각은 했었는데 더 길어질 것 같아서 어떻게 해야하나 고민하다가 그냥 뒀었는데 따로 변수를 선언해서 하면 깔끔하게 보일 것 같네요 수정하겠습니다!